docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
Note that we haven't added these headers yet, so if they exist, they were sent to us. I agree it doesn't make sense for the sender to send them, but does it make sense for us to remove them?